charon/temperature-control/temperature-control.ino

```diff
@@ -27,41 +34,65 @@ const int LOOP_DELAY = 500; // ms
 // We want the algae to stay at a certain temperature
 const float setpoint = 10.0; // °C
 
+// variable to store temperature tolerance for implementing hysteresis
+float setpoint_hysteresis = 0.5; // °C
+
+// variable to track duration of peltier on and off-time
+int cooling_duration = 0;
```

Comment:
nit: this is a bit misleading, since it also tracking the time when the peltier is off
nit^2: add `ms` unit comment?

Reply:
these are both very valid and the second actually makes it conform to how the comments are styled elsewhere, which is good